provider/postgis/postgis.go

```diff
@@ -659,7 +684,7 @@ func (p Provider) TileFeatures(ctx context.Context, layer string, tile provider.
 	}
 
 	now := time.Now()
-	rows, err := p.pool.Query(sql)
+	rows, err := p.pool.Query(context.Background(), sql)
```

Comment:
You have access to the actual context here, so we should use that instead of `context.Background()`